bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/TriggerGCService.java

```diff
@@ -68,9 +68,9 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
                 bookieServer.getBookie().getLedgerStorage().forceGC();
             } else {
                 @SuppressWarnings("unchecked")
-                Map<String, Object> configMap = JsonUtil.fromJson(requestBody, HashMap.class);
-                Boolean forceMajor = (Boolean) configMap.getOrDefault("forceMajor", null);
-                Boolean forceMinor = (Boolean) configMap.getOrDefault("forceMinor", null);
+                Map<String, String> configMap = JsonUtil.fromJson(requestBody, HashMap.class);
```

Comment:
If the `requestBody` doesn't pass a String, the JsonUtil.fromJson will throw an exception. 
Would you please add a test for this change?

Reply:
@hangc0276 Thanks for the review. I will add the respective tests.